juju/juju
# Allow message when suspending relation; suspnd/resume more than 1

## Description of change

The suspend/resume relation CLI gains the ability to suspend/resume more than one relation, plus add a message to relation status (on the offering side) when suspending a relation.

As a driveby, fix exports for models with remote applications with relations. This fix allows juju dump-model to work.

## QA steps

Deploy a cmr scenario.
Suspend remote relation(s) with a message
Look at juju offers --format yaml and check the status message on the offer
Ensure that juju dump-model works with suspended relations.
